tests/test_csc.py

```diff
@@ -160,9 +160,17 @@ async def test_chiller_alarm_turns_lamp_off(self):
                 topic=self.remote.evt_errorCode,
                 errorCode=0,
             )
+            await self.assert_next_sample(
+                topic=self.remote.evt_lampState,
+                basicState=LampBasicState.UNKNOWN,
+                controllerState=LampControllerState.UNKNOWN,
+                controllerError=LampControllerError.UNKNOWN,
+            )
```

Comment:
Would it be possible to assert the `lamp_on_time` and `lamp_off_time` as well? Perhaps only to verify that one is larger than the other after switching on of off the lamp.

Reply:
Good suggestion. I'll work at it.